test/topology/suite.yaml

```diff
@@ -1,4 +1,6 @@
 type: Topology
+# Relative load of this suite, higher values will be scheduled first
+suite_load: 20
```

Comment:
It's not very clear how to correctly select the load. We could use the concept of shares  like in seastar: we assign each suite some number of shares, then we sum all shares of all suites, and this gives us relative priority of each suite. What do you think?

Reply:
The problem with shares is you have to rebalance it. I think we don't need such precise accounting as long as long-running suites are started earlier.